app.go

```diff
@@ -461,6 +467,25 @@ func (app *App) StopTimeout() time.Duration {
 	return app.stopTimeout
 }
 
+func (app *App) visualizeDOT() DotGraph {
+	// Memoize the graph
+	if app.dotgraph != nil {
+		return *app.dotgraph
+	}
+	b := &bytes.Buffer{}
```

Comment:
style nit: use `var` form for zero value of buffer.

    var buff bytes.Buffer